pkg/loop/internal/pb/ccipocr3/reportcodec.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+
+option go_package = "github.com/smartcontractkit/chainlink-common/pkg/loop/internal/pb/ccipocr3;ccipocr3pb";
+
+package loop.internal.pb.ccipocr3;
+
+import "models.proto";
+
+// CommitPluginCodec is a service that can encode and decode CommitPluginReport.
```

Comment:
This is a good comment but lets also add when this is implemented (in this case I think its per family).